src/modules/twitterDb.js

```diff
@@ -1,7 +1,7 @@
 import { firestore } from './firebase';
 
-export const emojiExists = async emoji => {
-  let docRef = firestore.collection('emojis').doc(emoji);
+export const emojiExists = async (emoji, networkSuffix = '') => {
+  let docRef = firestore.collection('emojis' + networkSuffix).doc(emoji);
```

Comment:
I added the network suffix so that these emojis don't get mixed up in the database. (for example :gorilla: could be a different project on the mainnet and on the testnet)

Reply:
Smart, and needed.